storefront/app/controllers/workarea/storefront/pages_controller.rb

```diff
@@ -19,5 +19,11 @@ def robots; end
     def accessibility; end
     def browser_config; end
     def web_manifest; end
+
+    private
+
+    def page_specific_content_policy
```

Comment:
Delegation would be less lines of code!

Reply:
in the controller, but I'd have to define a method elsewhere. Don't think it's worth it.